templates/terraform/decoders/disk.erb

```diff
@@ -5,9 +5,12 @@ if v, ok := res["diskEncryptionKey"]; ok {
   transformed["rawKey"] = d.Get("disk_encryption_key.0.raw_key")
   transformed["sha256"] = original["sha256"]
   <% unless version.nil? || version == 'ga' %>
-  // The response for crypto keys often includes the version of the key which needs to be removed
-  // format: projects/<project>/locations/<region>/keyRings/<keyring>/cryptoKeys/<key>/cryptoKeyVersions/1
-  transformed["kmsKeyName"] = strings.Split(original["kmsKeyName"].(string), "/cryptoKeyVersions")[0]
+  if v, ok := original["kmsKeyName"]; ok {
```

Comment:
I would recommend using something other then `v` so you don't create shadow variables.